applications/serial_lte_modem/src/mqtt_c/slm_at_mqtt.c

```diff
@@ -459,6 +473,55 @@ static int do_mqtt_subscribe(uint16_t op,
 	return err;
 }
 
+/* Handles AT#XMQTTCFG commands. */
+int handle_at_mqtt_config(enum at_cmd_type cmd_type)
```

Comment:
@junqingzou: Since we have moved from dynamic configuration to respecting the static configuration in UART as well as with GNSS (well, not fully), I have to ask, what is the value for all customers in allowing dynamic configuration for CONFIG_MQTT_KEEPALIVE and CONFIG_MQTT_CLEAN_SESSION?

Reply:
It's out of the scope of the PR but I could explain a bit. It was actually what I used in testing against the Chinese MQTT broker to find the best time of Keep Alive that would not lose TCP connection in China Mobile network. Each carrier network has different firewall and NAT TCP binding so one  static Keep Alive will not work in all networks. Furthermore, MQTT broker has so-called max keep-alive parameter, which is not fetchable in MQTT 3.1.x but can in MQTT 5.x. Dynamic MQTT keep-alive on the client side allows adjusting for both needs of above.

Reply:
Wen't through the usage of CONFIG_MQTT_KEEPALIVE and CONFIG_MQTT_CLEAN_SESSION and beyond using hard coded values in nrf_cloud, there did not seem to be any reason why this could not be allowed.

Reply:
Zephyr MQTT "hardcode" [here](https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/net/lib/mqtt/mqtt.c#L179) and expect app to modify after init().